Examples/Framework/src/Framework/Sequencer.cpp

```diff
@@ -394,6 +394,14 @@ struct TimingInfo {
   DFE_NAMEDTUPLE(TimingInfo, identifier, time_total_s, time_perevent_s);
 };
 
+// struct to store timing data
+struct EventTimingInfo {
+  std::string identifier;
+  double time_s = 0;
+  int eventId;
```

Comment:
```suggestion
  double time_s = 0;
  int eventId = 0;
```

I would either initialize both or none of both